docs/topics/native/native-cocoapods-dsl-reference.md

```diff
@@ -52,18 +52,21 @@ You can use the following blocks, functions, and properties inside it:
 
 ### Targets
 
-* `ios`
-* `osx`
-* `tvos`
-* `watchos`
+| iOS                 | macOS        | tvOS                 | watchOS                 |
+|---------------------|--------------|----------------------|-------------------------|
+| `iosX64`            | `macosX64`   | `tvosX64`            | `watchosX64`            |
+| `iosArm64`          | `macosArm64` | `tvosArm64`          | `watchosArm64`          |
+| `iosSimulatorArm64` |              | `tvosSimulatorArm64` | `watchosSimulatorArm64` |
+|                     |              |                      | `watchosArm32`          |
+|                     |              |                      | `watchosDeviceArm64`    |
 
 For each target, use the `deploymentTarget` property to specify the minimum target version for the Pod library.
 
 When applied, CocoaPods adds both `debug` and `release` frameworks as output binaries for all of the targets.
 
 ```kotlin
 kotlin {
-    ios()
+    iosX64()
```

Comment:
Should we use maybe a `iosArm64` as a default? `iosX64` is slowly going away I think, as x86 macs are not available anymore

Reply:
That makes sense, thank you! Updated throughout the docs